ui-manchette-with-spacetimechart/src/helpers.ts

```diff
@@ -10,28 +10,29 @@ import type {
 
 import { BASE_OP_HEIGHT, MAX_TIME_WINDOW, MAX_ZOOM_X, MIN_ZOOM_X } from './consts';
```

Comment:
We should rename it as well

```suggestion
import { BASE_WAYPOINT_HEIGHT, MAX_TIME_WINDOW, MAX_ZOOM_X, MIN_ZOOM_X } from './consts';
```